src/main/java/jep/python/PyCallable.java

```diff
@@ -91,8 +106,23 @@ public Object call(Object... args) throws JepException {
      *             if an error occurs
      */
     public Object call(Map<String, Object> kwargs) throws JepException {
+        return callAs(Object.class, kwargs);
+    }
+
+    /**
+     * Invokes this callable with keyword args.
+     * 
+     * @param expectedType
+     *            The expected return type of the invocation
+     * @param kwargs
+     *            a Map of keyword args
+     * @return an {@link Object} value
```

Comment:
You are inconsistent on the @return javadoc for the callAs. They should all be "a value of the given type"

Reply:
:+1: fixed